src/app/components/listing/details/criteria/criteria.jsx

```diff
@@ -26,16 +26,26 @@ import {
   qmsStandard,
 } from 'shared/prop-types';
 
-const useStyles = makeStyles({
-  NestedAccordionLevelOne: {
-    borderRadius: '8px',
-    display: 'grid',
-  },
-  NestedAccordionLevelOneSummary: {
-    backgroundColor: '#f9f9f9',
-    borderRadius: '8px',
-  },
-});
+const useStyles = makeStyles(() => ({
+    infoIconColor: {
+      color: '#156dac',
+      marginLeft: '4px',
+      marginTop: '4px',
+    },
+    NestedAccordionLevelOne: {
+      borderRadius: '4px',
+      display: 'grid',
+      borderColor: ' #c2c6ca',
+      borderWidth: '.5px',
+      borderStyle: 'solid',
+    },
+    NestedAccordionLevelOneSummary: {
+      backgroundColor: '#EFEFEF!important',
```

Comment:
```suggestion
      backgroundColor: '#efefef !important',
```

There should be a space before `!important`, and our hex codes should all be lowercase